src/Controls/src/Core/Platform/Android/Extensions/FormattedStringExtensions.cs

```diff
@@ -64,6 +64,12 @@ public static SpannableString ToSpannableString(this Label label)
 
 			var builder = new StringBuilder();
 
+			var fontMetrics = context?.Resources?.DisplayMetrics != null ? new Paint()
+			{
+				TextSize = TypedValue.ApplyDimension(ComplexUnitType.Sp, (float)defaultFontSize, context.Resources.DisplayMetrics)
+			}.GetFontMetrics()
+			: null;
```

Comment:
Could we write this code in Java, it looks like it interops between C# and Java a lot. I've seen some past issue complaining about Android FormattedString performance like:

* https://github.com/dotnet/maui/issues/14222

So we'd make a version of this method that returns "Font metrics" instead of "Window metrics":

https://github.com/dotnet/maui/blob/ad28b65309edbfa3be0e01480662ee5cb013b5d4/src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformInterop.java#L581-L592

Reply:
Yes, that would be beneficial, but I'm not sure how to write this code in JAVA and how to use it in MAUI

Reply:
If you add a method in this file, it will magically appear on the `PlatformInterop` static class, you can call from C# like:
```
//signature
public static Rect GetCurrentWindowMetrics(Activity activity);

//usage
var rect = PlatformInterop.GetCurrentWindowMetrics(myActivity);
```

Reply:
Thank you! I pushed a new commit with this functionality

Reply:
I made one small change, but it seems like this should be ready if we fix up the screenshot.